### PR TITLE
sql: make example in RBAC doc easier to understand

### PR DESCRIPTION
**Pull request by ireneontheway**
### What is changed, added or deleted? (Required)

To close [#2496](https://github.com/pingcap/docs-cn/issues/2496)
make example easier to understand.
user can try RBAC by simply copy and paste sql in doc.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/2570
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by TomShawn** [comment]
@ireneontheway Looks like https://github.com/pingcap/docs-cn/pull/2570 is a bit antiquated. It is recommended to update the document according to the latest upstream version. Otherwise, outdated content might be aligned.

File: role-based-access-control.md [hunk 1]
```diff
@@ -35,60 +35,55 @@ Roles are stored in the `mysql.user` table. The name of the role you are trying
...
 The operation of granting a privilege to a role is the same with that of granting a privilege to a user. For details, see [TiDB Privilege Management](/privilege-management.md).
 
-For example, you can use the following statement to grant the `analyst` role the privilege to read the `test` database:
+For example, you can use the following statement to grant the `app_read` role the privilege to read the `app_db` database:
 
 {{< copyable "sql" >}}
 
 ```sql
-GRANT SELECT ON test.* TO 'analyst'@'%';
+GRANT SELECT ON app_db.* TO 'app_read'@'%';;
```
Comment (ireneontheway) [inline role-based-access-control.md:47]:
```suggestion
GRANT SELECT ON app_db.* TO 'app_read'@'%';
```

File: role-based-access-control.md [hunk 2]
```diff
@@ -224,129 +283,91 @@ SET ROLE ALL EXCEPT 'role1', 'role2' --
...
++----------------+
+| `app_read`@`%` |
++----------------+
+```
 
-You can use the `SHOW GRANTS` statement to check which role has been granted to the user.
+### Revoke a role
 
-To check privilege-related information of another user, you need the `SELECT` privilege on the `mysql` database.
+For example, you can use the following statement to revoke all privileges on the `app_read` role granted to users `read_user1@localhost` and the `read_user2@localhost`:
```
[the remark this answers is not in the document]
Comment (ireneontheway) [reply role-based-access-control.md:327]: > 解除角色 `app_read` 与用户 `read_user1@localhost`、`read_user2@localhost` 的授权关系。

from the source text

[the remark this answers is not in the document]
Comment (TomShawn) [reply role-based-access-control.md:327]: But the title is to `Revoke a role`, not to `revoke privileges` which is in the subsequent section.

**Review by imtbkcat — APPROVED** [review]
LGTM

File: role-based-access-control.md [hunk 3]
```diff
@@ -224,129 +283,90 @@ SET ROLE ALL EXCEPT 'role1', 'role2' --
...
-+---------------------------------------------------------------------+
-| GRANT USAGE ON *.* TO `u1`@`localhost`                              |
-| GRANT Select, Insert, Update, Delete ON `db1`.* TO `u1`@`localhost` |
-| GRANT `r1`@`%`,`r2`@`%` TO `u1`@`localhost`                         |
-+---------------------------------------------------------------------+
+For details, see [TiDB Privilege Management](/privilege-management.md).
+
+### Delete a role
+
+For example, you can use the following statement to delete roles `app_read` and `app_write`:
```
Comment (TomShawn) [inline role-based-access-control.md:359]:
```suggestion
You can use the following statement to delete roles `app_read` and `app_write`:
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**